rootfs/rootfs/etc/rc.d/automount-shares

```diff
@@ -6,7 +6,7 @@ set -e
 # VirtualBox Guest Additions
 # - this will bail quickly and gracefully if we're not in VBox
 if modprobe vboxguest &> /dev/null && modprobe vboxsf &> /dev/null; then
-	mountOptions='defaults'
+	mountOptions='iocharset=utf8'
```

Comment:
@tianon I have read the usage info,and tested the default iocharset.But,it didn't work,so I added it.